platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/State.java

```diff
@@ -72,8 +75,35 @@ default void unregisterCommitListener(@NonNull final StateChangeListener listene
         throw new UnsupportedOperationException();
     }
 
+    /***
+     * {$inheritDoc}
```

Comment:
Nit: shouldn't this be `@`? 

```suggestion
     * {@inheritDoc}
```

Reply:
yes, fixed